storage/database/db_migration.go

```diff
@@ -53,45 +54,52 @@ func (dbm *databaseManager) StartDBMigration(dstdbm DBManager) error {
 
 	// vars for log
 	start := time.Now()
-	iterateNum, fetchedTotal, fetched := 0, 0, 1
+	cycleNum, fetchedTotal, previousFetchedTotal, fetched := 0, 0, 0, 1
 
 loop:
 	for fetched > 0 {
```

Comment:
How to break this for loop in normal case?

Reply:
If there is no fetch items, `fetched` is equal to 0 and it is able to escape the loop.

Reply:
This is useless? Log statements and sigQuit statements can be included in main loop?

Reply:
review reflected PTAL :)